components/ResultTable/ResultTable.js

```diff
@@ -0,0 +1,38 @@
+export default ({ players, set, matchWinner }) => {
+  return (
+    <table>
+      <thead>
+        <tr>
+          {Array.apply(null, Array(set.bestOf + 1)).map((item, i) => (
+            <th>{i === 0 ? '' : i}</th>
+          ))}
+        </tr>
+      </thead>
+      <tbody>
+        {players
+          .sort(p => !(p.id === matchWinner))
```

Comment:
nit: use full word in iterables, for instance:
`players.sort(player => player.id !== matchWinner)`
or use destructuring:
`players.sort(({ id }) => id !== matchWinner)`

Reply:
👍 good convention. I'll change it